build/patches/Disable-unified-autoplay-feature.patch

```diff
@@ -1,23 +0,0 @@
-From: csagan5 <32685696+csagan5@users.noreply.github.com>
-Date: Sat, 19 Dec 2020 20:04:14 +0100
-Subject: Disable unified autoplay feature
-
----
- media/base/media_switches.cc | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
-
-diff --git a/media/base/media_switches.cc b/media/base/media_switches.cc
---- a/media/base/media_switches.cc
-+++ b/media/base/media_switches.cc
-@@ -480,7 +480,7 @@ const base::Feature kUseR16Texture{"use-r16-texture",
- // Enables the Unified Autoplay policy by overriding the platform's default
- // autoplay policy.
- const base::Feature kUnifiedAutoplay{"UnifiedAutoplay",
--                                     base::FEATURE_ENABLED_BY_DEFAULT};
-+                                     base::FEATURE_DISABLED_BY_DEFAULT};
```

Comment:
Can you remind me what the unified autoplay does exactly?

Reply:
set the policy `kDocumentUserActivationRequired`
```
// The document needs to have received a user activation or received one
// before navigating.
```
that is, any click on the document

instead of `kUserGestureRequired`
```
// A local user gesture on the element is required.
```
that is, a click on the element.

Reply:
Ok; and you want to enable it because the local user gesture will be replaced by the permissions request instead.